.mdox.yaml

```diff
@@ -0,0 +1,201 @@
+version: 1
+
+inputDir: "docs"
+outputDir: "website/docs-pre-processed/tip"
+extraInputGlobs:
+  - "CHANGELOG.md"
+  - "SECURITY.md"
+  - "MAINTAINERS.md"
+  - "CONTRIBUTING.md"
+  - "CODE_OF_CONDUCT.md"
+
+gitIgnored: true
+localLinksStyle:
+  hugo:
+    indexFileName: "_index.md"
+
+transformations:
+
+  - glob: "../CHANGELOG.md"
+    path: /thanos/CHANGELOG.md
+    frontMatter:
+      template: |
+        title: Changelog
+        type: docs
+        menu: thanos
+        lastmod: "{{ .Origin.LastMod }}"
+    backMatter:
+      template: |
+        Found a typo, inconsistency or missing information in our docs?
+        Help us to improve [Thanos](https://thanos.io) documentation by proposing a fix [on GitHub here](https://github.com/thanos-io/thanos/edit/main/{{ .Origin.Filename }}) :heart:
+
+  - glob: "../MAINTAINERS.md"
+    path: /thanos/MAINTAINERS.md
+    frontMatter:
+      template: |
+        title: Maintainers
+        type: docs
+        menu: thanos
+        lastmod: "{{ .Origin.LastMod }}"
+    backMatter:
+      template: |
+        Found a typo, inconsistency or missing information in our docs?
```

Comment:
We could try https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/ (: